### ISSUE TITLE
editor button shows up in terminal accessible buffer

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#191813]
I get the following bug:

1. disable screenreader mode.
2. set `terminal.integrated.focusAfterRun: true`
3. highlight text in launch.json and then run `Terminal: Run selected text in active terminal`
4. Strange terminal state, `Add configuration` in terminal

<img width="940" alt="image" src="https://github.com/microsoft/vscode/assets/31675041/9d63d023-ae39-4540-ba00-6944ee53225c">

_Originally posted by @andreamah in https://github.com/microsoft/vscode/issues/189776#issuecomment-1699489858_

**Comment by meganrogge** [comment]
i cannot reproduce this. Please reopen if you can @andreamah

**Comment by andreamah** [comment]
This doesn't seem to happen anymore 👍
